INSTALL.md

```diff
@@ -121,33 +113,71 @@ clang and blocks support:
 	./configure CC=clang --with-blocks-runtime=/usr/local/lib
 	make check
 
-IV. Building and installing for Linux
+### Building and installing for Linux
 
 Note that libdispatch development and testing is done only
 on Ubuntu; currently supported versions are 14.04, 15.10 and 16.04.
 
-(1) The first thing to do is install required packages:
+1. The first thing to do is install required packages:
+    
  1a. Install build tools and clang compiler.
-    sudo apt-get install autoconf libtool pkg-config clang
+    `sudo apt-get install autoconf libtool pkg-config clang`
+ 
  1b. Install dtrace (to generate provider.h)
-    sudo apt-get install systemtap-sdt-dev
+    `sudo apt-get install systemtap-sdt-dev`
+ 
  1c. Install additional libdispatch dependencies
-    sudo apt-get install libblocksruntime-dev libkqueue-dev libbsd-dev
+    `sudo apt-get install libblocksruntime-dev libkqueue-dev libbsd-dev`
 
-Note: compiling libdispatch requires clang 3.8 or better and
+    Note: compiling libdispatch requires clang 3.8 or better and
 the gold linker. If the default clang on your Ubuntu version is
 too old, see http://apt.llvm.org/ to install a newer version.
 On older Ubuntu releases, you may need to install binutils-gold
 to get the gold linker.
 
-(2) Initialize git submodules.
+2. Initialize git submodules.
   We are using git submodules to incorporate specific revisions of the
   upstream pthread_workqueue and libkqueue projects into the build.
+
+    ```
     git submodule init
     git submodule update
+    ```
+    
+3. Build (as in the general instructions above)
 
-(3) Build (as in the general instructions above)
-	sh autogen.sh
-	./configure
-	make
+    ```
+    sh autogen.sh
+    ./configure
+    make
     make install
+    ```
+
+#### Building toolchain in Linux
```

Comment:
I think it actually makes sense for this section to be moved up under the "Configuring and installing libdispatch (general comments)" section as it follows on from how to build libdispatch into an existing Swift toolchain.

It should be marked that this is Linux only - but it might be worth stating that's because Swift on macOS platforms uses the system installed libdispatch, so its not required.